workflows/prognostic_c48_run/runtime/config.py

```diff
@@ -123,3 +127,15 @@ def get_model_urls(config_dict: dict) -> List[str]:
         for entry in prephysics_config:
             urls += entry.get("model", [])
     return urls
+
+
+def get_wrapper(config: UserConfig):
+    """Import the appropriate wrapper based on the config"""
+    module = f"{config.fortran_model}.wrapper"
```

Comment:
Are we guaranteed these modules will always have the format `{fortran_model}.wrapper`? I guess realistically we have control here and it's not that likely we'll have more than two.

Reply:
Switched to this approach in d510c3b.